feature.go

```diff
@@ -0,0 +1,237 @@
+package lago
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+)
+
+type FeatureRequest struct {
+	client *Client
+}
+
+type PrivilegeConfig struct {
+	SelectOptions []string `json:"select_options,omitempty"`
+}
+
+type Privilege struct {
+	Code      string          `json:"code"`
+	Name      string          `json:"name,omitempty"`
+	ValueType string          `json:"value_type"` // TODO: add enum
```

Comment:
If you don't add the enum now, it will be a breaking change later on.

Reply:
This isn't marked as `required` in the OpenAPI.

Reply:
updated both